src/applications/gi/sass/partials/_gi-profile-page.scss

```diff
@@ -181,11 +181,11 @@
   .contact-heading {
     vertical-align: top;
     margin: 0;
+    font-family: 'Source Sans Pro';
```

Comment:
I think it would be better to apply [`.vads-u-font-family--sans`](https://design.va.gov/utilities/font-family) to the items with the `contact-heading` class.